addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1650,7 +1715,9 @@ var SnippetsMenu = Widget.extend({
      * @param {OdooEvent} ev
      */
     _onActivateInsertionZones: function (ev) {
+        this._insertDropzoneForbidSanitize = ev.data.forbidSanitize;
```

Comment:
Same here, add a comment that it should be a new param in master but was done that way in stable? Or I am misunderstanding something here?